Source/VaRestPlugin/Private/VaRestJsonObject.cpp

```diff
@@ -713,17 +713,9 @@ bool UVaRestJsonObject::WriteToFile(const FString& Path)
 	return true;
 }
 
-bool UVaRestJsonObject::WriteToFilePath(const FString& Path, bool bIsRelativePath)
+bool UVaRestJsonObject::WriteToFilePath(const FString& Path, const FString& BasePath)
 {
-	if (bIsRelativePath)
-	{
-		FString FullJsonPath = FPaths::ConvertRelativePathToFull(FPaths::ProjectDir() / Path);
-		return WriteToFile(FullJsonPath);
-	}
-	else
-	{
-		return WriteToFile(Path);
-	}
+	return WriteToFile(BasePath / Path);
```

Comment:
It's equal to `WriteToFilePath(BasePath / Path, false)`. Also breaking `FPaths::ProjectDir` automatic usage.